docs/en_US/Compression/CompressionUtils.rst

```diff
@@ -144,6 +144,22 @@ When the masks of different layers in a model have conflict (for example, assign
    from nni.compression.pytorch.utils.mask_conflict import fix_mask_conflict
    fixed_mask = fix_mask_conflict('./resnet18_mask', net, data)
 
+not_safe_to_prune
+^^^^^^^^^^^^^^^^^
+
+If we try to prune a layer whose output tensor is taken as the input by a shape-constraint OP(for example, view, reshape), then such pruning maybe not be safe. For example, we have a convolutional layer followed by a view function.
+
+.. code-block:: bash
+
+   x = self.conv(x) # output shape is (batch, 1024, 3, 3)
+   x = x.view(-1, 1024)
+
+If the output shape of the pruned conv layer is not disivible by 1024(for example(batch, 500, 3, 3)), we may meet a shape error. We cannot replace such a function that directly operates on the Tensor. Therefore, we need to be careful when pruning such layers. The function not_safe_to_prune finds all the layers followed by a shape-constraint function. Here is an example for usage.
+
+.. code-block:: bash
+
+   not_safe = not_safe_to_prune(model, dummy_input)
```

Comment:
do we have some suggestions for the users when they detect the not-safe layers?

Reply:
Good suggestion!